lib/logic.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module TicTacToe
+  class Game
+    attr_accessor :player1, :player2, :board
+    def initialize(first_player, second_player)
+      @player1 = first_player
+      @player2 = second_player
```

Comment:
There should be a separated Player class, please make it instead of putting everything inside the Game class.

Reply:
Resolved. Thanks